decidim-core/config/locales/en.yml

```diff
@@ -267,6 +267,7 @@ en:
         ignore: |-
           If you don't want to accept the invitation, please ignore this email.<br />
           Your account won't be created until you access the link above and set your password.
+        invited_you: "%{invited_by} has invited you to %{application}, you can accept it through the link below."
```

Comment:
Correction: `%{invited_by} has invited you to %{application}. You can accept the invitation through the link below:`

Reply:
This is not even used lol 🎉